pac4j-oidc/src/main/java/org/pac4j/oidc/profile/OidcProfile.java

```diff
@@ -55,6 +56,14 @@ public JWT getIdToken() throws ParseException {
         return null;
     }
 
+    public String getRefreshTokenString() {
```

Comment:
I think it can't become a JWT. An example of refresh_token is `13/FuJRLB-4xn_4rd9iJPAUL0-gApRRtpDYuXH5ub5uW5Ne0-oSohI6jUTnlb1cYPMIHq0Ne63h8HdZjAidLFlgNg==` by [Dex](https://github.com/coreos/dex)  and `25/3Uv_Sp3komLpvJTPHUr4bnTE3J-HkpECwOdXmpG-hk8MEudVrK5jSpoR30zcRFq6` by Google. I tried both in [jwt.io](http://jwt.io/) without luck.